File: MVC_O_DiceGame/DiceGame_csharp/DiceGame/view/SweConsole.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace DiceGame.view
{
	class SweConsole : Console
	{
		override public void DisplayInstructions()
		{
			System.Console.Clear();
			String message = "Välkommen till tärningsspelet. Tryck en tangent för att spela, eller s för att sluta";
			System.Console.WriteLine(message);
		}

		override public bool WantsToPlay()
		{
			return System.Console.ReadKey().KeyChar != 's';
		}

		override public void DisplayResult(bool a_isWinner)
		{
			if (a_isWinner)
			{
				System.Console.WriteLine("Du är en vinnare!");
			}
			else
			{
				System.Console.WriteLine("Ledsen men du förlorade!");
			}
		}
	}
}

File: MVC_O_DiceGame/DiceGame_ruby/view/sweConsole.rb
# coding: utf-8
module View
  class SweConsole < Console

    QUIT_ON_INPUT  = "s"

    def display_instructions
      system "clear"
      puts "Välkommen till tärningsspelet. Tryck en tangent för att spela, eller #{QUIT_ON_INPUT} för att sluta"
    end

    def wants_to_quit?
      gets == "#{QUIT_ON_INPUT}\n"
    end

    def display_result player_won
        if player_won
          puts "Du är en vinnare!"
        else
          puts "Ledsen men du förlorade"
        end
    end

  end
end
